application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/messages/internal/Messages.java

```diff
@@ -648,13 +649,15 @@ public static String siblingMessage(int itemsInSiblings, int nSiblings) {
 		return NLS.bind(message, itemsInSiblings, nSiblings);
 	}
 
-	public static String stackTraceMessage(double duration, double totalDuration, String frameFraction) {
+	public static String stackTraceMessage(
+		double duration, String durationUnit, double totalDuration, String totalDurationUnit, String frameFraction) {
 		String message;
 		if (duration == 1) {
```

Comment:
- You didn't change thus part if the code but I think this could be written `1.0` or `1d` as `duration` is a double.